syntax = "proto3";

package agent.connect.authorize.v1;

import "google/protobuf/duration.proto";

// [#protodoc-title: Authorize]

// [#proto-status: experimental]
message AuthorizePayload {
    string Target = 1;
    string ClientCertURI = 2;
    string ClientCertSerial = 3;
}
